src/api/client.ts

```diff
@@ -0,0 +1,12 @@
+import AsyncStorage from "@react-native-async-storage/async-storage"
+import { createClient } from "urql"
+
+export const client = createClient({
+    url: 'https://api-pr-100-zpsq.onrender.com/graphql',
```

Comment:
I don't know why this still works, the `/graphql` has been changed to just the root (`https://api-pr-100-zpsq.onrender.com/`)

Reply:
Oh I added it by default and it worked. I can change it. This is a temporary URL so it would need to replaced with the actual loop API url once that's ready.